src/objects/zcl_abapgit_objects_generic.clas.abap

```diff
@@ -691,4 +710,26 @@ CLASS zcl_abapgit_objects_generic IMPLEMENTATION.
     ENDIF.
 
   ENDMETHOD.
+
+  METHOD apply_clear_logic.
+    DATA lv_table TYPE tabname.
+
+    IF mo_field_rules IS BOUND.
+      lv_table = iv_table.
+      mo_field_rules->apply_clear_logic( EXPORTING iv_table = lv_table
+                                         CHANGING  ct_data  = ct_data ).
+    ENDIF.
+  ENDMETHOD.
```

Comment:
this should do the trick with 2 lines less 

```abap
   IF mo_field_rules IS BOUND.
      mo_field_rules->apply_clear_logic( EXPORTING iv_table = |{ iv_table }|
                                         CHANGING  ct_data  = ct_data ).
    ENDIF.
```

Reply:
I was bit worried to use latest syntax due the restriction. Good to know this works and fixed.